QMLComponents/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ target_include_directories(
 target_link_libraries(
   QMLComponents
   PUBLIC
+         Common
```

Comment:
Aaah that really should have been there before yeah, I guess we werent using anything from it yet though. As it was compiling fine.

But good idea to avoid some future issues ^^

Reply:
This already generates linking issues for me currently

Reply:
I figured ^^